devops/helm/clam-av/_clamav/configurations/1.0/clamd.conf

```diff
@@ -562,7 +562,7 @@ MaxRecursion 10
 # Note: disabling this limit or setting it too high may result in severe damage
 # to the system.
 # Default: 10000
-MaxFiles 100
+MaxFiles 10000
```

Comment:
As it is set to default value please comment this line leaving the default value to be in effect.